pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1219,10 +1219,6 @@ func GetPodAffinityTerms(podAffinity *v1.PodAffinity) (terms []v1.PodAffinityTer
 		if len(podAffinity.RequiredDuringSchedulingIgnoredDuringExecution) != 0 {
 			terms = podAffinity.RequiredDuringSchedulingIgnoredDuringExecution
 		}
-		// TODO: Uncomment this block when implement RequiredDuringSchedulingRequiredDuringExecution.
```

Comment:
We should keep this. `RequiredDuringSchedulingRequiredDuringExecution` hasn't been implemented yet.